Add Monthly Beatmapping Contest and update Contests (#3139)
ppy/osu-wiki: wiki/Contests/Monthly_Beatmapping_Contest/en.md

# Monthly Beatmapping Contest

As implied by its name, the Monthly Beatmapping Contest (_MBC_) is a contest that runs monthly and allows users to compete with their beatmapping capabilities. 

The MBC was originally managed on a nearly monthly basis between 2013 and 2015 and was followed by hiatus until 2020. The first 12 iterations of the MBC can be found on the [Contests](/wiki/Contests#monthly-beatmapping-contest) article. The results for the 13th iteration can be found [here](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results).

> Since 2020 the MBC has adopted a new schedule, ruleset, voting procedure, and reward scheme. The rest of this article details the current format for Monthly Beatmapping Contests.

Reviewer: ```suggestion
Since 2020, the MBC has adopted a new schedule, ruleset, voting procedure, and reward scheme. The rest of this article details the current format for Monthly Beatmapping Contests.
```

## Schedule

The timeline for each month's contest is as follows:

- **Beatmapping (3 weeks)**: Contest is announced at the start of the month and beatmappers create their entries.
- **Screening (1 week)**: Entries are evaluated by experienced beatmappers.
- **Voting (2 weeks)**: Top entries from screening are published for the osu! community to vote on.
- **Rewards**: Winners are announced and prizes are redeemed.

The voting portion of each contest overlaps with the next month's contest's beatmapping portion.

## Ruleset

Submissions for the MBC must abide by the following rules:

> - **Submissions must follow the month's beatmapping limitation.** The limitation differs month by month and is detailed on each contest's announcement post.

Reviewer: Is this point about submitting multiple times using different songs? I was going to ask a similar question, but if so, this would answer it.

Author: it's a gimmick for the contest, like "your map must use circles only"
unrelated to songs or multiple submissions

> - **Submissions must be a single difficulty for one the month's selected songs.** Selected songs differ month by month and are included in each contest's announcement post.

Reviewer: ```suggestion
- **Submissions must be a single difficulty for one of the month's selected songs.** Selected songs differ month by month and are included in each contest's announcement post.
```
?

Reviewer: ```suggestion
- **Submissions must be a single difficulty from one of the month's selected songs.** Selected songs differ month by month and are included in each contest's announcement post.
```
?

- **Submissions must abide by the [ranking criteria](/wiki/Ranking_Criteria).**

One or more rules may be added or dropped depending on the month's specifications. Any exceptions will be clearly explained in the contest's announcement post.

## Results

Results for the MBC are first narrowed down to the most competitive entries, then published for community vote. Creators of all entries are anonymous in the screening and voting stages.

### Screening

A group of at least 10 experienced beatmappers (*screening crew*) are tasked with reviewing all entries and providing their top picks in order of preference. Entries that receive the most praise from the screening crew will move on to the voting stage.

### Voting

The amount of beatmaps that advance to the voting stage varies depending on the amount of highly competitive submissions the contest receives and will usually range between 5 and 10 finalists.

Voting takes place on the respective contest's [Community Contests page](https://osu.ppy.sh/community/contests). Users are encouraged to vote for their favorite entry and to look out for any entries that effectively work with the month's beatmapping limitation.

> ### Previous contests

Reviewer 2: feels weird that this isn't its own level 2 header

Author: moved it, hopefully better

Information and results for all iterations of the MBC since its relaunch in 2020 can be found below.

| Contest Listing | Winner | Runner-up | Screening crew's #1 | Results Post |
| :-- | :-- | :-- | :-- | :-- |
| [January 2020](https://osu.ppy.sh/community/contests/63) | TBD | TBD | TBD | TBD |

## Rewards

The Monthly Beatmapping Contest rewards users with points for different types of results (*MBC points*):

- **Community vote winner:** 2 points
- **Community vote runner-up:** 1 point
- **Screening crew's #1 pick:** 1 point

MBC points reward users at certain thresholds:

- **Profile badge:** badges are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
- ***Elite Mapper* forum title:** attainable at 10 points.

> - **osu!supporter:** each point earned provides a user with a month of supporter for themself or someone of their choosing.

Reviewer 2: should be written as normal sentences instead of with a colon, with the current wording. like the first could be
```markdown
- **Profile badges** are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
```
